Installation-CentOS-8-Apache

```diff
@@ -0,0 +1,365 @@
+source: Installation/Installation-CentOS-8-Apache.md
+path: blob/master/doc/
+
+> NOTE: These instructions assume you are the **root** user.  If you
+> are not, prepend `sudo` to the shell commands (the ones that aren't
+> at `mysql>` prompts) or temporarily become a user with root
+> privileges with `sudo -s` or `sudo -i`.
+
+**Please note the minimum supported PHP version is 7.2.5**
+
+**This installation guide will install the [Dispatcher Service](https://docs.librenms.org/Extensions/Dispatcher-Service/) for a standalone installation by default**
+
+## Install Common Required Packages ##
+
+```
+dnf install epel-release
+```
+Enable `PowerTools` repo to be able to install the `whois` package
+
+```
+dnf config-manager --set-enabled PowerTools
+dnf install python3 git cronie fping whois ImageMagick mtr python3-PyMySQL net-snmp net-snmp-utils nmap python3-memcached rrdtool python3-policycoreutils httpd mariadb mariadb-server unzip policycoreutils-python-utils
+pip3 install python-dotenv
+```
+To fix compatibility issues with hardcoded python paths, create a symlink for python3
+
+```
+ln -sf /usr/bin/python3 /usr/bin/python
```

Comment:
This doesn't seem right.

Reply:
It is because some scripts like ./scripts/github-remove does not want to run unless you explicitly put python3 in front of it. The script just references 'python'. Check [here](https://www.techsupportpk.com/2019/12/how-to-install-and-configure-librenms-centos-rhel8.html) where this user had the same thing, only difference is he used python3 binary explicitly. I know you are dropping support for python2, but until then, this workaround will help solve a lot of confusion.

Reply:
if you do this you'll eventually run into problems with python script running deeper in system

Reply:
Then I think we need to hold off on the merge of the PR until the porting to Python3 by LNMS is done. I will remove this in the meantime, but it should not be merged until all references to 'python' is converted to 'python3'.

Reply:
could you give me the Traceback of a Python Script which doesn't want to run?

linking python3 against python is "invalid" and possibly dangerous.
Many Scripts from distribution Maintainers are written in python and often still Python2.

Default is still python2 if version is not declared. Thinking this will be still the next years.
If you explicit want to run as script in python3 call it with "python3 -scriptname-".
If it's an executable script write in filehead "#!/usr/bin/env python3" and it's executes with python3

Reply:
From what I understood from the [announcement](https://community.librenms.org/t/dropping-support-for-php-7-1-and-python-2/11851/2) all script in LNMS will from now on be run with 'python3' command and not just 'python'. Hence my suggestion to hold off on the PR till the end of the month, then there will no confusion. 
I fully understand that creating a link may not be a good idea. So don't get me wrong, off that idea now. All I am saying is we are probably better off to wait with the merge of this PR then, till end of the month when all is updated and called with python3.

Reply:
Correct they will. But therefore also Python scripts are/will be converted to be Python3 compatible.
if you give me a traceback of your "doesn't want to run" i can eventually tell you what's wrong or missing :-)
I don't see a reason for waiting to merge your PR into master till Python conversion is done.

Reply:
Destroyed this machine, will rebuild and run all the install stuff again, then provide you with one. In the meantime, it was the `./scripts/github-remove` script. Will get you a proper error code in the next few hours.

Reply:
For a start:
![image](https://user-images.githubusercontent.com/24249616/80963265-e02db680-8e0e-11ea-9720-848c84c29c3d.png)

But also:
https://pastebin.com/r8TRPZfW

Sorry, but strace is the only way I know how to make any form of traceback. If you want something else, please let me know?

Reply:
what happens if you write in shell `python --version`

Reply:
![image](https://user-images.githubusercontent.com/24249616/80965012-21739580-8e12-11ea-8a34-ba8a6ddce441.png)

Note, I DID NOT create the symlink in this instance. So this is a "fresh" install.

Reply:
jo ... python2 is not installed ;-)

Reply:
CentOS8. I know Python2 is not installed. It was sort-of the idea behind the whole write-up.

Reply:
mmh interesting. It's the first distribution without Python2 by default in it.
Nethertheless Python Scripts are pointing with "#!/usr/bin/env python3" to python3. Thats platform independent.
For yourself you could link "python" to python3, but without warranty everything works fine.
From Python2 to Python3 are many changes, cleanups and cutoffs made, so old Python scripts may be partial incompatible if not written to work on 2 and 3 ;-)
But good to know CentOS8 comes without Python2, thx for the time and information :+1:

Reply:
No need for me to link anything, as the exercise was purely to get an official CentOS 8 installation instruction in the docs. If LNMS will explicitly call `python3` as indicated, then I will this PR as-is now, until the updates have been made at the end of the month. Alternatively, let me know what else needs to be fixed if you want to merge it now, so that I can do it for you.